manifests/server/account_security.pp

```diff
@@ -1,14 +1,22 @@
 class mysql::server::account_security {
   mysql_user {
-    [ "root@${::fqdn}",
-      'root@127.0.0.1',
+    [ 'root@127.0.0.1',
       'root@::1',
-      "@${::fqdn}",
       '@localhost',
```

Comment:
root@localhost should not be removed. That's the main account which is used if you connect over a UNIX socket, which is the default.